### PR TITLE
adds a transportation/vehicle sector to the data parser

### LINKED ISSUE #116: Add vehicle sector in data_parser
Currently, the transportation sector is counted in the industrial sector. Transportation should be its own sector. 

This issue can be closed when the ``make_plots`` function in ``data_parser.py`` produces plots for each sector
* industrial
* electrical
* transport

### PR DESCRIPTION
**Pull request by samgdotson**
This PR adds a vehicle transportation sector in the data parser file. 
Closes #116

### CODE REVIEW

**Comment by robfairh** [comment]
I think the implementation is correct.
However, I got the following error when running 8th model.

[err.txt](https://github.com/arfc/pride/files/5666628/err.txt)

Also, some of the plots are named properly and some others are not.

image

**Comment by samgdotson** [comment]
Ah, this is because snakemake is looking for these files but the script no longer produces all of them! I'll fix this in snakemake

**Comment by samgdotson** [comment]
@robfairh This should be fixed. I stopped trying to be clever in Snakemake and just made it specific. It should work now.

File: temoa-uiuc/Snakefile [hunk 1]
```diff
@@ -13,7 +13,7 @@ DBS = ["08_uiuc"]
 SCENARIOS = ["scenario08"]
 FOLDER = ["08_uiuc_scenario08_model"]
 
-SECTORS = ["ind", "elc", "all"]
+SECTORS = ["ind", "elc", "vcl","all"]
```
Comment (robfairh) [inline temoa-uiuc/Snakefile]:
```suggestion
SECTORS = ["ind", "elc", "vcl", "all"]
```

**Review by robfairh — APPROVED** [review]

**Comment by robfairh** [comment]
good!

**Pull request merged**